src/Entity/User.php

```diff
@@ -894,4 +897,14 @@ public function canUpdateUser(User $actor): bool
             return $this->apDomain === $actor->apDomain;
         }
     }
+
+    public function getRelatedLinks(): array
```

Comment:
Could you add phpDoc so we know which kind of array is returned? I really liked if the returned value would be of `RelatedLink[]` so an array of objects, so we can add some validation logic so we can mark links as "checked" or something so we know that the rel=me is actually correct

Reply:
Discussed Discussed on element chat.
I don't know a elegant way to denormalize the json field from table and return it on get method. This'll required some dependencies inside entity that I think is not a good practice.
Instead, I created a RelatedLinkDTO class and this will be returned in UserDTO.